src/main/resources/expected_input.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Is there a reason this is in the `src/main/resources` directory, instead of the `src/test/resources` directory?